tests/core/caching-utils/test_generate_cache_key.py

```diff
@@ -3,14 +3,14 @@
 from eth_utils import (
     to_dict,
 )
-from hypothesis import (
-    given,
-    strategies as st,
-)
 
 from web3._utils.caching import (
     generate_cache_key,
 )
+from web3.tools.hypothesis import (
```

Comment:
This is still importing from `web3` where it should be importing directly from `hytophesis`.